tuf/utils/pkcs8.go

```diff
@@ -0,0 +1,299 @@
+package utils
```

Comment:
Did we end up copying this from somewhere else and is there a license we should be preserving?